localstack/services/events/scheduler.py

```diff
@@ -25,7 +61,10 @@ def run(self):
 
     def should_run_now(self):
         schedule = CronTab(self.schedule)
-        delay_secs = schedule.next()
+        delay_secs = schedule.next(
+            default_utc=True
+        )  # utc default time format for rule schedule cron
+        # TODO fix execute on exact cron time
```

Comment:
the rework of the job scheduler in v2 needs to deal with executing a schedule cron at the exact specified time, not run the job if it should occur in less than 60 seconds from now.